code/metrics.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+
+
+def hits_at_k(y_true, y_pred=None, k=None, is_l2=False):
```

Comment:
I think hits@k should be defined from both ends. We should know how well we are doing in finding if mails are written by sender, recv and how well are predicting suspicious mails